src/Service/CountryService.php

```diff
@@ -45,4 +45,11 @@ public static function isPostNLShipmentFromNLToBE(AbstractConsignment $consignme
             && MyParcelBE::getModule()->isNL()
             && CarrierPostNL::NAME === $consignment->getCarrierName();
     }
+
+    public static function isPostNLToEU(AbstractConsignment $consignment): bool
```

Comment:
Deze checkt of het niet NL of BE is, ik denk dat de naam van de method daarom te verwarrend is. Ook VS zou hier true returnen, terwijl het niet in de EU ligt. Ik snap waar het vandaag komt, maar voor sanity zou ik toch de methodnaam aanpassen.